Source/LibationCli/Options/SearchOptions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using ApplicationServices;
+using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibationCli.Options;
+
+[Verb("search", HelpText = "Search for books in your library")]
+internal class SearchOptions : OptionsBase
```

Comment:
Interesting. I never considered that search might be useful from CLI.

Reply:
When I added options to specify asins for `liberate`, `convert`, and `set-status` the question of how are users going to find those asins naturally led to including a search.